data/user_data_base.ps2

```diff
@@ -389,9 +415,15 @@ $attributesContent = $attributesContent + @"
     Write-Output "Beginning write to file"
     [System.IO.File]::WriteAllLines($attributeFile, $attributesContent)
     Write-Output "Chef attributes saved"
-    $env:Path += ";C:\opscode\chef\bin"
     Write-Output "Starting Chef run."
-    Start-Process -FilePath "chef-client" -ArgumentList "-c `"c:\chef\client.rb`" -r `"role[$chef_role]`" -L `"c:\chef\chef-client-initial-run.log`" -j `"c:\chef\attributes.json`"" -Wait -NoNewWindow
+
+    # Create a script for starting chef on disk then run it.
```

Comment:
so if chef fails we can just rerun c:\hudl\run-chef.ps1 rather than having to know the command to run it.